squarelet/users/models.py

```diff
@@ -217,3 +217,38 @@ def is_hub_eligible(self):
             | Q(groups__hub_eligible=True)
             | Q(parent__hub_eligible=True)
         ).exists()
+
+
+class LoginLog(models.Model):
```

Comment:
Do we need a plan to truncate this table at some point when we have two billion logins?

Reply:
I think ideally we would have somewhere we can put this data where we can easily analyze it.  Is that just here and we build some visualizations on top?  Possibly try putting it in datasette?  Or use a third party tool that would let us analyze it?  I am also not sure if we want to just truncate after a certain amount of time or if we want to archive old data for historical purposes